.github/workflows/ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Pipeline
+on: [push]
```

Comment:
pls restrict the trigger to
- open PR against master branch
- pushes against a specific branch. So let's say you develop feature/X - you may not want to trigger the build (as it's a work in progress) until it's officially merged into develop branch.